source/iostreams.tex

```diff
@@ -1173,8 +1177,8 @@
 The format control information for both input and output.
 \end{itemdescr}
 
-\indexlibrary{\idxcode{fmtflags}!\idxcode{ios_base}}%
 \indexlibrary{\idxcode{ios_base}!\idxcode{fmtflags}}%
+\indexlibrary{\idxcode{fmtflags}!\idxcode{ios_base}}%
```

Comment:
Is this entry correct? It looks like the function is called `flags`, no` fmtflags`?!